lib/rustler_precompiled.ex

```diff
@@ -454,6 +454,13 @@ defmodule RustlerPrecompiled do
 
         %{target_system | arch: arch, vendor: vendor}
 
+      target_system.os =~ "freebsd" ->
+        arch = normalize_linux_arch(target_system.arch)
+
+        vendor = with "portbld" <- target_system.vendor, do: "unknown"
```

Comment:
I'm pretty sure we will want to include more vendors here in the future. But let's go with this one first :)